rfcs/2021-07-27-7117-vrl-string-interpolation.md

```diff
@@ -177,6 +172,16 @@ F-strings could be fallible and their use could require any errors to be handled
 f'The date is { .date: %v %R }' ?? "invalid date"
 ```
 
+### Output error text
+
+Rather than forcing the user to handle errors, if an error occurs the error text is output. For example:
```

Comment:
I really don't think this is good UX given the alternatives discussed. When would a user want an error message sprawled into a string somewhere, without even knowing that it occurred?